File: program9.java
import java.util.Scanner;

class program9 
{
    //there should not be input output in function
    //no user interaction in function
    float AddTwoNumbers (
                            float fNo1 ,                            // fisrt input
                            float fNo2                              // second input
                        )
    {
        float fAns = 0.0f;

        fAns = fNo1 +fNo2;                                           // perform addition

        return fAns;
    }
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        float fValue1 =0.0f , fValue2 = 0.0f , fResult = 0.0f;      // variable creation with default values

        System.out.println("Enter First Number : ");               // accept first number
        fValue1 = sobj.nextFloat();

        System.out.println("Enter Second Number : ");             // accept second number
        fValue2 = sobj.nextFloat();

        program9 pobj = new program9();                             // create object of class to call non static function

        fResult = pobj.AddTwoNumbers(fValue1 , fValue2);            // perform addition by calling function and storing result in fResult   

        System.out.println("Addition is : "+fResult);               //print the result

        sobj.close();
        
    }
}

File: program9.cpp
#include<iostream>
using namespace std;

//there should not be input output in function
//no user interaction in function

float AddTwoNumbers(
                        float fNo1 ,            // To accept 1st number as No1
                        float fNo2              // To accept 2nd number as No2
                    )
{
    float fAns = 0.0f;                          // To store the result of Addition of No1 and No2

    fAns = fNo1 + fNo2;                         // Perform Addition of No1 and No2

    return fAns;                                // Return the result to the caller
}

int main()
{
    float fValue1 = 0.0f;                           // To store 1st input
    float fValue2 = 0.0f;                           // To store 2nd input
    float fResult = 0.0f;                           // To store the Result

    cout<<"Enter First Number :";                   // Accept 1st number as No1
    cin>>fValue1;

    cout<<"Enter Second Number :";                  // Accept 2nd number as No2
    cin>>fValue2;

    fResult = AddTwoNumbers(fValue1 , fValue2);      // Perform Addition of No1 and No2

    cout<<"Addition is : "<<fResult;                // Display the result

    return 0;
}